server/spec/cards.spec.js

```diff
@@ -0,0 +1,28 @@
+describe("Post request /api/v1/cards", () => {
```

Comment:
@datarockets/cuties What do you think about tests folders structure?

Kind of:
```
spec/
  support/
    helpers/
  models/
  repositories/
  features/ (controllers)
  services/ 
  system/
```
?

So `spec/cards.spec.js` would be `spec/contollers/cards.spec.js`